css/style.css

```diff
@@ -1,114 +1,53 @@
 *{
-margin: 0;
-padding: 0;
-font-family: Century Gothic;
+  margin: 0;
+  padding: 0;
 }
 
-header
-{
-background-image:linear-gradient(rgba(0,0,0,0.5),rgba(0,0,0,0.5)),url(../7.jpg);
-height:100vh;
-background-size:cover;
-background-position:center;
-
-}
-.title{
-
-position:absolute;
-text-align:center;
-top:50%;
-left:50%;
-transform:translate(-50%,-50%);
-}
-
-.title h1{
-color:  #fefe10;
-font-size:40px;
-font-weight: bold; 
-}
-ul{
-float:right;
-list-style-type:none;
-margin-top:25px;
-}
-ul li
-{
-display:inline-block;
-}
-ul li a {
-text-decoration:none;
-color:#fff;
-padding:5px 20px;
-border: 1px solid transparent;
-transition:0.6s ease
-}
-ul li a:hover{
-background-color:powderblue;
-color:#000;
-}
-.main{
-max-width: 1200px;
-margin:auto;
-}
-ul li.active a{
-background-color:powderblue;
-color:#000;
-}
-
-.search-box{
-position:absolute;
-top:65%;
-left:50%;
-transform:translate(-50%,-50%);
-background:powderblue;
-height:40px;
-border-radius:40px;
-padding:10px;
-justify-content:center;
-align-items:center;
-}
-.search-box:hover > .search-txt{
-width:240px;
-padding:0 6px;
+body{
+  background-image: url("../images/7.jpg");
+  background-size:cover;
+  background-position:center;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
 
-.search-box:hover > .search-btn{
-background:#000;
+input{
+  height: 50px;
+  font-size: 1.2em;
+  width: 0;
+  outline: none;
+  padding-left: 10px;
+  border: none;
+  border-radius: 15px;
+  transition: all .5s;
 }
-.search-btn{
-color:#e84118;
-float:right;
-width:40px;
-height:40px;
-border-radius:50%;
-background:#2f3640;
-display:flex;
-justify-content:center;
-align-items:center;
-transition:0.4s;
-}
-.h1 {
-  height: 100%;
-  width: 100%;
-  
 
-  align-items: center;
+.icon{
+  height: 60px;
+  width: 60px;
+  background: red;
+  border-radius: 50%;
+  display: flex;
   justify-content: center;
+  align-items: center;
+  font-size: 1.8em;
+  color: #fff;
+  cursor: pointer;
+  transform: translateX(-30px);
 }
 
-.search-txt
-{
-border:none;
-background:none;
-outline:none;
-float:left;
-padding:0;
-color:black;
-font-size:16px;
-transition:0.4s;
-line-height:40px;
-width:0px;
+.icon span{
+  transform: rotate(-45deg);
 }
 
+.active{
+  width: 60%;
+}
 
-
+input:hover{
+  box-shadow: 0 0 5px #fff,
+      0 0 10px #fff,
+      0 0 15px #fff;
+}
```

Comment:
I think I had already discussed the styling matters into the html file with their importance... bcz it's already there I also expect that do necessary changes at there in the single file so that it would be beneficial for you as well (not to change in multiple files)

Reply:
Okayyy